orbita.php

```diff
@@ -11,7 +11,7 @@
  * Plugin Name:     Órbita
  * Plugin URI:      https://gnun.es
  * Description:     Órbita é o plugin para criar um sistema Hacker News-like para o Manual do Usuário
- * Version:         1.2
+ * Version:         1.21
```

Comment:
@rghedin na verdade eu acabei de ver que fiz certo fazendo errado.

O vesionamento funciona assim:

1. - É a versão do plugin geral, ou seja, a primeira versão que fizemos, se um dia criarmos um novo, esse vira 2.
.12. - É a major, a versão de alteração maior, quando tem alguma atualização maior, aumentamos esse número em 1
.0 - É a minor, para correção de bugs ou alterações menores


A gente estava na versão `1.11.1`, e como foi feita uma alteração maior, devemos ir para a `1.12.0` ou `1.12`, agora vamos para `1.12.1`

```suggestion
 * Version:         1.12.1
```

Reply:
https://semver.org/lang/pt-BR/